cloudflare.md

```diff
@@ -1,22 +0,0 @@
-This CloudFlare app adds a simple text message at the top of your website if the visitor's browser version is too old.
```

Comment:
This is now set in the app configure page.
<img width="994" alt="screen shot 2017-03-16 at 18 45 20" src="https://cloud.githubusercontent.com/assets/592134/24025628/c3c07d74-0a78-11e7-980a-11941b993a4c.png">